portfolio/src/main/webapp/script.js

```diff
@@ -12,6 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const URL = "https://teachablemachine.withgoogle.com/models/8A06Q2Ycp/";
+
+let model, webcam, labelContainer, maxPredictions;
+
+// load image model, set up webcam
+async function predictImage() {
+  const modelURL = URL + "model.json";
+  const metadataURL = URL + "metadata.json";
+
+  // load model, metadata
+  model = await tmImage.load(modelURL, metadataURL);
+  // Model can predict what is seen by webcam
+  // either as Squidward or Spongebob
+  maxPredictions = model.getTotalClasses();
+
+  // set up webcam
+  const flip = true; // whether to flip the webcam
+  webcam = new tmImage.Webcam(200, 200, flip);
+  await webcam.setup(); // request access to the webcam
+  await webcam.play();
+  window.requestAnimationFrame(loop);
+
+  // append webcam and prediction to DOM
+  document.getElementById("webcam-container").appendChild(webcam.canvas);
+  labelContainer = document.getElementById("image-prediction");
+  for (let i = 0; i < maxPredictions; i++) {
+    labelContainer.appendChild(document.createElement("div"));
+  }
+}
+// better/more "meaningful" name for loop()?
```

Comment:
is `// better/more "meaningful" name for loop()?` meant to be a reminder to yourself? if so, you can do 
`TODO(jennyagao): find better/more "meaningful" name for loop()`

to actually address your comment it might be better to rename to `updateAndPredict()`

Reply:
can remove `// better/more "meaningful" name for loop()?` from your code